cmd/user-agent/src/pages/webWalletSamples.js

```diff
@@ -99,6 +105,41 @@ const udc = {
     }
 }
 
+const govnVC = {
```

Comment:
note: this is a governance agency credential, not the credential that the issuer/verifier would be holding.